File: SeqBootCamp/Day1/p3.c
#include <stdio.h>

//3 6 9 12 18 21 24 27

void main()
{
    int n, sum = 0;
    scanf("%d", &n);
    for (int i = 1;; i++)
    {
        int num = i * 3;
        if (num > n)
            break;
        if (num % 15 == 0)
            continue;
        printf("%d\n", num);
    }
}

File: SeqBootCamp/Day3/Yash/p3.js
// 1, 4, 7, 12, 23, 42, 77, ...N

let a = 1;
let b = 4;
let c = 7;

let sum = 0;
console.log(a);
console.log(b);
console.log(c);
let n = 200;
while (sum <= n) {
  sum = a + b + c;
  console.log(sum + " ");
  a = b;
  b = c;
  c = sum;
}
